### ISSUE TITLE
"Cannot read property 'process' of undefined" if --parser=flow flag

### ISSUE THREAD

**Issue by KirillYoYo** [reactjs/react-codemod#219]
Have a errors 
`flybridge\app\components\shared\charts\linechart\LineChartBrush.js Transformation error (Cannot read property 'process' of undefined)
TypeError: Cannot read property 'process' of undefined
`
then start `jscodeshift -t ./transforms/React-PropTypes-to-prop-types.js ../flybridge/app/components --parser=flow`

![image](https://user-images.githubusercontent.com/26794095/53415344-d3375b00-39f2-11e9-94f0-089dd70ba649.png)

**Comment by step-off** [comment]
Have the same issue while processing React component file with `--parser=flow` option.
@KirillYoYo did you find solution?

**Comment by KirillYoYo** [comment]
@step-off No...

**Comment by threepointone** [comment]
Do you have a sample file I can reproduce this with?

**Comment by step-off** [comment]
You can try to run any transformation with `--parser=flow` on this file : https://github.com/skbkontur/retail-ui/blob/master/packages/retail-ui/components/ComboBoxOld/ComboBoxOld.js
It should output error described above. If not, I'll gladly give you more specifics to reproduce :)

**Comment by threepointone** [comment]
I was not able to reproduce this, but I have some guesses - 
- it's an older version of jscodeshift that could be subtly broken
- there's a known issue with the way jscodeshift could fail on paths with '..' in them 
- the flow-parser module being used is somewhat off?

I've made some changes recently, so if you can try this and let me know: 
- be in the root folder of your source (say in `flybridge`)
- run `npx react-codemod React-PropTypes-to-prop-types app/components --dry` (it'll ask you to pick a parser, pick "JavaScript with Flow"). This command uses a --dry flag so it won't actually change your files. 
- if there are no errors, try it without the --dry flag. 

Let me know if that helps, cheers.

**Comment by akaustav** [comment]
Turns out that babel is the culprit. Using `--no-babel` as an argument to `jscodeshift` does the trick.
Thanks to @tulasimcabca for this comment: https://github.com/facebook/jscodeshift/issues/299#issuecomment-456468109

I was able to execute the following command:
```console
jscodeshift --no-babel --parser=babel --t /path/to/react-codemod/transforms/class.js /path/to/src
```

Maybe `react-codemod` should support passing this flag into jscodeshift until this issue is fixed.
